<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>decryption_form</class>
 <widget class="QDialog" name="decryption_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>decifra un testo</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci messaggio cifrato:
Vai a capo dopo ogni lettera!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_enc_mex">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>221</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decifra">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DECIFRA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_dec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_alfabeto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancella">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELLA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Se vuoi una mano, scegli da qua sotto le lettere
 per comporre il messaggio da decifrare</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
